lib/ojs.ml

```diff
@@ -113,3 +113,6 @@ let is_null x =
 
 let obj_type x =
   string_of_js (call (pure_js_expr "Object.prototype.toString") "call" [|x|])
+
+let internal_require = pure_js_expr "require"
+let require name = apply internal_require [| string_to_js name |]
```

Comment:
I'm not sure if adding `require` to gen_js_api  is too specific for the library purposes (convert from/to JS). There are plenty of ways of including 3rd party modules in JavaScript. Now Node supports [`import`](https://nodejs.org/api/esm.html#esm_import_specifiers) too, browsers can load `<script>` tags, etc.

 I started writing bindings that would provide the `require` call on behalf of user, to save them some time and trouble. And quite quickly [got feedback](https://github.com/jchavarri/jsoo-react/issues/47) from folks that didn't need `require` but instead preferred loading with `<script>` tags. In the end I just removed all 3rd party loading assumptions from the library and let the users decide how they want to import JS modules.

I guess it's not a big deal, but wanted to share this piece of context for what it's worth. Maybe it's worth to add some documentation about ways to reference 3rd party JavaScript loaded libs.

(As a side note, I still see the `Critical dependency: the request of a dependency is an expression` that I mentioned [here](https://github.com/LexiFi/gen_js_api/issues/66#issuecomment-526844888), when loading this code with Webpack when Dune builds is in `dev` profile)

Reply:
agreed ! let's remove Ojs.require and use a javascript files to execute the imports in the test/example.